pkg/scheduler/objects/application.go

```diff
@@ -1128,6 +1135,7 @@ func (sa *Application) tryReservedAllocate(headRoom *resources.Resource, nodeIte
 }
 
 func (sa *Application) tryPreemption(reserve *reservation, ask *AllocationAsk) bool {
+	ask.SetLastPreemptionAttempt(time.Now())
```

Comment:
We should not set this unless we have really preempted otherwise it looks like we did but we really did not. See comment above.

Reply:
Removed.